src/utils/encode.ts

```diff
@@ -39,13 +58,33 @@ export function stringToArrayBuffer(str: string): Uint8Array {
 
 /**
  * Convert string to array buffer (browser and node compatible)
+ *
+ * @param {string} a The Base64 encoded string to convert.
+ * @returns {Uint8Array} The decoded Uint8Array.
+ *
+ * @example
+ * ```typescript
+ * const base64String = 'SGVsbG8='; // 'Hello' in Base64
+ * const byteArray = atobUniversal(base64String);
+ * console.log(byteArray); // Outputs: Uint8Array {0: 72, 1: 105, 2: 108, 3: 108, 4: 111}
```

Comment:
Response is not an object, but an array : 
```
[ 72, 101, 108, 108, 111 ]
```

Reply:
![image](https://github.com/starknet-io/starknet.js/assets/81040730/12985fd1-0d53-45a1-818e-4fe093589795)